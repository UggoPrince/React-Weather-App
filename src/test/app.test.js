import React from 'react';
import renderer from 'react-test-renderer';
import App from '../App';

const title = 'React with Webpack and Babel';

describe('My Test Suite', () => {
  it('My Test Case', () => {
    expect(true).toEqual(true);
  });
});

describe('App', () => {
  test('snapshot renders', () => {
    const component = renderer.create(<App title={title} />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
