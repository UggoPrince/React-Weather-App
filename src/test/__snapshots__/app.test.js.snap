// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App snapshot renders 1`] = `
<div>
  React with Webpack and Babel
</div>
`;
